gradle/libraries.toml

```diff
@@ -95,7 +95,7 @@ gr8 = { group = "com.gradleup", name = "gr8-plugin", version = "0.8" }
 # to avoid leaking the Kotlin stdlib on the classpath
 #
 # Keep in sync with MIN_GRADLE_VERSION
-gradle-api-min = { group = "dev.gradleplugins", name = "gradle-api", version = "5.6" }
+gradle-api-min = { group = "dev.gradleplugins", name = "gradle-api", version = "6.8" }
```

Comment:
6.8 is the first Gradle version with Kotlin 1.4 support. 

Interestingly, Kotlin 1.3 can read 1.6 metadata but fails on `@JvmDefault` interfaces because there's also a per-class `versionRequirements` ([gist](https://gist.github.com/martinbonnin/c46a4b44fd3c4d5db6e737cc60dedd4e))